neoload/commands/test_results.py

```diff
@@ -201,6 +201,13 @@ def get_end_point(id_test: str = None, operation=''):
     slash_id_test = '' if id_test is None else '/' + id_test
     return rest_crud.base_endpoint_with_workspace() + __endpoint + slash_id_test + operation
 
+def prompt_is_locked(field):
```

Comment:
This method can be renamed into "prompt_boolean" in case it is reused later.

Reply:
yes its true !